integration_tests/pages/refer/showPerson.ts

```diff
@@ -0,0 +1,13 @@
+import Page from '../page'
+import type { Person } from '@accredited-programmes/models'
+
+export default class ShowPersonPage extends Page {
```

Comment:
Trying to figure out if we could reuse the original show person page here, but I think as it's technically a separate page this probably makes sense to duplicate. We _could_ do some inheritance from a common `PersonPage`-style class but I think that's probably overdoing it for this fairly simple page.

Reply:
Yeah - with the view, the only duplication was the summary list, and that's using a component anyway. Feels like they're separate pages, just with the same main content